sym-lib-table

```diff
@@ -1,4 +1,3 @@
 (sym_lib_table
-  (lib (name jasmine-rescue)(type Legacy)(uri ${KIPRJMOD}/jasmine-rescue.lib)(options "")(descr ""))
-  (lib (name gsg-symbols)(type Legacy)(uri ${KIPRJMOD}/gsg-kicad-lib/gsg-symbols.lib)(options "")(descr ""))
```

Comment:
I think this old gsg-symbols path should be restored, replacing the absolute path.